src/js/store/appSettings/reducers.ts

```diff
@@ -36,7 +36,8 @@ const initialState: AppSettings = {
       widgetId: 'aa',
       uiParams: [{}]
     }
-  ]
+  ],
+  iso: 'CAF'
```

Comment:
does CAF mean specific country? can we have this as an empty string to be less confusing for the default state?